advocacy_docs/partner_docs/VeritasGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring NetBackup for PostgreSQL'
+---
+
+
```

Comment:
I like the section of info we have in Thales and Nutanix above Prerequisites.  Can we do the same?

Reply:
Section info added